appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/src/main/java/org/kie/appformer/formmodeler/codegen/view/impl/java/inputs/util/SourceGenerationValueConvertersFactory.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.appformer.formmodeler.codegen.view.impl.java.inputs.util;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kie.workbench.common.forms.model.FieldDefinition;
+
+public class SourceGenerationValueConvertersFactory {
```

Comment:
Could you please add a javadoc or comment here that explains why we can lookup a converter with only one class name? I think intuitively one would expect that you lookup a converter with two types (the domain and target types). So it would be good to document why each type you lookup only has a single other type it converts to.